frontend/src/v5/store/users/users.helpers.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { clientConfigService } from '@/v4/services/clientConfig';
+import { generateV5ApiUrl } from '@/v5/services/api/default';
+
+export const getMemberImgSrc = (teamspace: string, member: string) => (
+	generateV5ApiUrl(`teamspaces/${teamspace}/members/${member}/avatar?${Date.now()}`, clientConfigService.GET_API)
```

Comment:
What is the date for?

Reply:
If a user changes their avatar the image won't update as it will have cached the old picture and use that. Adding the date like this ensures that it makes a new request and so the latest avatar is used. I'm wondering if this should be turned off when used in the assignees component as there are a lot of avatars, therefore a lot of requests and up-to-date avatars might be less important. @sanmont3drepo what do you think?

Reply:
Is not important for now

Reply:
I don't really understand why we do this to begin with, it's not vital for the user to have the latest avatar so we're just imposing extra requests to the server for not a lot of benefit. If anything it encourages confusion if we have 2 different looking avatars within the same view because the user updated in between the requests